module/applications/components/damage-application.mjs

```diff
@@ -206,7 +206,29 @@ export default class DamageApplicationElement extends HTMLElement {
 
     // Calculate damage to apply
     const targetOptions = this.getTargetOptions(uuid);
-    const { total } = this.calculateDamage(token, targetOptions);
+    const { total, active } = this.calculateDamage(token, targetOptions);
+
+    const types = [];
+    for ( const [change, values] of Object.entries(active) ) {
+      for ( const type of values ) {
+        const config = CONFIG.DND5E.damageTypes[type] ?? CONFIG.DND5E.healingTypes[type];
+        if ( !config ) continue;
+        const data = { type, change, icon: config.icon };
+        types.push(data);
+      }
+    }
+    const changeSources = types.reduce((acc, {type, change, icon}) => {
+      const { label, pressed } = this.getChangeSourceOptions(type, change, targetOptions);
+      acc += `
+        <button class="change-source unbutton" type="button" data-type="${type}" data-change="${change}"
+                data-tooltip="${label}" aria-label="${label}" aria-pressed="${pressed}">
+          <dnd5e-icon src="${icon}" inert></dnd5e-icon>
+          <i class="fa-solid fa-slash inert"></i>
+          <i class="fa-solid fa-chevron-down" inert></i>
```

Comment:
There's not a lot of room to work with and I think you've done a great job with what is available. I found the chevron a little hard to see at first that it was a downwards chevron with some of the damage types I tested with (some were easier to see than others). We could try `angles-down` rather than `chevron-down` maybe. See what you think, if it doesn't get in the way of the icon too much. Otherwise we can stick with the chevron since it's good enough.